fv3core/stencils/map_single.py

```diff
@@ -75,6 +75,8 @@ def lagrangian_contributions(
                         * (q4_2 + 0.5 * esl * (q4_3 - q4_2 + q4_4 * (1.0 - r23 * esl)))
                         / (pbot - ptop)
                     )
+    with computation(FORWARD), interval(0, 1):
+        q2_adds = 0
```

Comment:
Using `0.0` here removes the implicit cast from int to float here. Not required, but it gives the impression that this is a float type explicitly.